redash/models/users.py

```diff
@@ -122,6 +123,10 @@ def __init__(self, *args, **kwargs):
             kwargs["email"] = kwargs["email"].lower()
         super(User, self).__init__(*args, **kwargs)
 
+    @property
+    def is_active(self):
```

Comment:
I didn't find this property used to.
My suggestion is that if this attribute is not used temporarily, I do not recommend adding it.

Reply:
OK